frontend/app/src/App.tsx

```diff
@@ -1717,6 +1729,7 @@ export class App extends PureComponent<Props, State> {
             onPageChange: this.onPageChange,
             currentPageScriptHash,
             libConfig,
+            currentFragmentId: this.state.currentFragmentId || "",
```

Comment:
same as [here](https://github.com/streamlit/streamlit/pull/8164/files#r1495062820) with regards to `||` vs `??`